### ISSUE TITLE
Is it possible that *.a_ctg.gfa represents sex chromosome? 

### ISSUE THREAD

**Issue by Jung19911124** [chhylp123/hifiasm#318]
Hi, 

I ran Hifiasm on Hifi reads derived from an inbred strain of XX/XY species. 
In order to determine Y chromosome sequences, I extracted genomic DNA from male.

The result file, *.p_ctg.gfa, included some fragments, which are not found in reference assembly derived from female (XX individual).
Those fragments must be linked to Y chromosome, but total length of the fragments was much smaller than I expected. 

Then I realized *.a_ctg.gfa also consists of the small fragments which are not found in the female genome, and have a reasonable length for Y chromosome. 

Is it possible that *.a_ctg.gfa represents heterogametic sex chromosome when we used an inbred strain?

Any advice would be appreciated.
Best,

Jung

**Comment by chhylp123** [comment]
Yes, it is possible and it also depends on how large the Y chromosome is.

**Comment by Jung19911124** [comment]
Dear chhylp123,

Thank you for the reply.
My species is an insect, so chromosome size is relatively small.
Y chromosome of this species is estimated to be < 40 Mbp.
